jujugui/static/gui/src/app/components/notification-list/notification-list.js

```diff
@@ -56,6 +56,19 @@ YUI.add('notification-list', function() {
         type: notification.level,
         timestamp: notification.timestamp
       };
+
+      // Emiting a google tag manager event registering the notification.
+      /* global dataLayer */
```

Comment:
the globals are defined in the `eslint.rc` file. With that said however `dataLayer` is quite generic, is there any way we can make this more unique? I'm guessing that this is defined somewhere out of our control as it's not defined in this PR?